app/models/katello/glue/pulp/repo.rb

```diff
@@ -786,6 +806,7 @@ def index_content
       self.index_db_docker_manifests
       self.index_db_puppet_modules
       self.index_db_package_groups
+      self.index_db_ostree_branches if Katello::RepositoryTypeManager.repository_types.key?(Repository::OSTREE_TYPE)
```

Comment:
Seems like a good method to add to RepositoryTypeManager would be to check if a type was enabled instead of relying on knowledge of the data structure they are stored in.